incident_scraper/external/google_nbd.py

```diff
@@ -28,6 +29,8 @@ def __init__(self):
     def _create_incident_from_dict(incident: dict):
         """Convert an incident dict to a Incident Model."""
         return Incident(
+            id=f"{incident['UCPD_ID']}_"
+            f"{incident['ReportedDate'].strftime(UCPD_MDY_KEY_DATE_FORMAT)}",
```

Comment:
Made the `UCPD ID` field along with the date to the ID, so that we do not get copies of the same report in the Datastore.